src/Commands/SpdxCommands.php

```diff
@@ -88,8 +91,26 @@ public function importLicenses(string $graph_uri, array $options = ['clean' => F
       ]));
     }
 
+    if (empty($licences)) {
+      $rdf_files_to_import = $rdf_files;
+    }
+    else {
```

Comment:
This was changed for the initial approach of the ticket and I thought that it might be useful to keep. I will revert the commit if you think it is unneeded.